Resources/Prototypes/Loadouts/Jobs/Heads/chiefjustice.yml

```
@@ -0,0 +1,36 @@
+- type: loadout
+  id: LoadoutClothingOuterRobesJudge
+  category: JobsCommand
+  cost: 0
+  exclusive: true
+  requirements: 
```

yamllint

[error] trailing spaces

(trailing-spaces)